src/components/dashboard/components/dictionary/ListDictionaries.jsx

```diff
@@ -1,39 +1,46 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import Card from './DictionaryCard';
-import Loader from '../../../Loader';
+import React from "react"
```

Comment:
This file has a lot of linting errors, no semicolons on import declarations, strings are supposed to use single quotes I don't know why you chose to use double quotes. Kindly integrate your linter in your editor

Reply:
I do have them installed on my editor